metadata.yaml

```diff
@@ -17,6 +17,7 @@ doc_title: "Auxiliary Telescope: Using CWFS during operations and for collimatio
 
 # Author names, ordered as a list. Each author name should be formatted as 'First Last'
 authors:
+  - "Patrick Ingraham"
   - "Tiago Ribeiro"
```

Comment:
Not sure I can be considered an author of this tech note in particular. Feel free to remove my name if you like.

Reply:
No. Authorship doesn't mean you have to write words!